apps/web/app/chatwoot/ChatwootWidget.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import { useCallback, useEffect, useRef } from "react";
+import { TUser } from "@formbricks/types/user";
+
+interface ChatwootWidgetProps {
+  chatwootBaseUrl: string;
+  isChatwootConfigured: boolean;
+  chatwootWebsiteToken?: string;
+  user?: TUser | null;
```

Comment:
we should only pass the user information to the frontend that we actually need here. As we only need id, name and email let's limit the props to this to not expose more personal information that we need to.